Feature tsbridge

pitschi opened the pull request:
Added tsbridge as another proxy to the project. It works with tailscale.com but should also work with headscale. I might add headscale to the project in the future. 

The structure is in parallel to all the other stacks, The README.md documents how to use it. It might have to be transferred into the documentation though. 

Had to adapt a line in the homepage stack. That might be more beautiful in the future. Other than that it can be seamlessly introduced into all the other stacks a similar way like traefik is. It is at this point not interfering with traefik or other mechanisms. Will put it in with all the stacks that I will do for my homelab. 

Please advise for any changes in structure or additional stuff you need.

pitschi commented:
Love what you are doing with the abstraction. #whoopwhoop

Tarow commented:
I think the abstraction works alright now.

The remaining issue is OIDC: When using tsbridge, it seems to setup a handler using tsnet for each service.
Multiple containers managed by tsbridge are not able to connect to each other by their tailscale domain name because the tsbridge proxy only affects ingress. Egress is not routed through the tailnet. 

The connection from containers like Gatus to Authelia by the issuer url is a requirement tho, currently Gatus, Outline and other OIDC supported apps fail to start up.

Don't have a solution for this yet.

pitschi commented:
two ways: either tsbridge is not used but ts is added as a sidecar. needs more resources ofc. but egress is then routed. 

Or: since the mechanisms need to reach the url, I activated funnel for the mechanism I use (authentik). Funnel exposes the service to the world on 443 (two other ports possible as well) with let's encrypt certs on the service name under the tsnet dns. Configuration of funnel node attribute and access rules needed on tailscale to make it happen. once configured, it is very convenient. funnel can then be activated by 

`services.podman.containers.MYSERVICE.tsbridge.funnelEnabled = true;`

 https://tailscale.com/kb/1223/funnel

Tarow commented:
Can you provide a minimal example of say `tsbridge`, `gatus` and `authelia` with OIDC enabled?

When the `reverseProxy.domain` is configured with `example.ts.net` and Authelia has the subdomain `auth`, then Gatus is configured with the issuer url `https://auth.example.ts.net`. During startup it will try to connect.

Now i see two problems, even with the extra host configured like this:
```yaml 
services:
  gatus:
    extra_hosts:
      - "auth.example.ts.net: 192.168.1.100"  # Authelia's homelab IP
```

1. The port would be wrong, as Authelia doesn't listen on 443
2. There'd be no https when not going through the tailnet, so the connection would also fail

I can't change the issuer-url for Gatus either, as that would break redirects for the user.
Also i am pretty sure that some applications require a secure context (https) for OIDC.

I'd be happy to see a working example though, i might be missing something.

renovate[bot] committed:
chore(deps): update beszel to v0.18.6 (#1278)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/linuxserver/healthchecks docker tag to v4.1.1-ls339 (#1282)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Tarow committed:
feat: add trek stack

* feat: add nomad stack

* fix: docs links

* feat: add oidc

* refactor: icons

* chore: env variable

* chore: update image

* docs: readme

renovate[bot] committed:
chore(deps): update docker.n8n.io/n8nio/n8n docker tag to v2.15.0 (#1283)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/gotenberg/gotenberg docker tag to v8.29.1 (#1284)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/grafana/alloy docker tag to v1.15.0 (#1285)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update kaneo to v2.4.4 (#1287)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v2.7.1 (#1286)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/binwiederhier/ntfy docker tag to v2.21.0 (#1288)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/gethomepage/homepage docker tag to v1.12.2 (#1289)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/deluan/navidrome docker tag to v0.61.0 (#1290)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/home-assistant/home-assistant docker tag to v2026.4.0 (#1293)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/alam00000/bentopdf-simple docker tag to v2.8.1 (#1292)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update lscr.io/linuxserver/jellyfin docker tag to v10.11.7 (#1291)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/rommapp/romm docker tag to v4.8.0 (#1294)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Tarow committed:
feat: add dawarich stack

* feat: dawarich WIP

* feat: oidc setup

* docs: update readme

renovate[bot] committed:
chore(deps): update docker.io/crowdsecurity/crowdsec docker tag to v1.7.7 (#1296)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/gethomepage/homepage docker tag to v1.12.3 (#1297)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Tarow committed:
feat(trek): oidc admin group support

Tarow committed:
fix(trek): create lldap admingroup

renovate[bot] committed:
chore(deps): update docker.io/freikin/dawarich docker tag to v1.6.1 (#1298)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update kaneo to v2.5.1 (#1300)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v2.7.2 (#1299)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/tandoorrecipes/recipes docker tag to v2.6.2 (#1301)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/prom/prometheus docker tag to v3.11.0 (#1302)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update lscr.io/linuxserver/radarr docker tag to v6.1.1 (#1303)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update kaneo to v2.5.2 (#1304)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/stirlingtools/stirling-pdf docker tag to v2.9.0 (#1305)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/papra-hq/papra docker tag to v26.4.0 (#1307)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/gtsteffaniak/filebrowser docker tag to v1.3.3 (#1306)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Tarow committed:
feat: add anchor stack

* feat: anchor WIP

* refactor: options

* docs: add github link

Tarow committed:
docs: add anchor to readme

renovate[bot] committed:
chore(deps): update ghcr.io/tandoorrecipes/recipes docker tag to v2.6.3 (#1309)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Tarow committed:
feat: add homebox module

* feat: homebox module

* fix: typo

Tarow committed:
refactor: authelia healthcheck

Tarow committed:
chore: update template flake version

renovate[bot] committed:
chore(deps): update ghcr.io/h44z/wg-portal docker tag to v2.2.3 (#1311)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/tandoorrecipes/recipes docker tag to v2.6.4 (#1312)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/home-assistant/home-assistant docker tag to v2026.4.1 (#1314)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/stirlingtools/stirling-pdf docker tag to v2.9.1 (#1313)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update kaneo to v2.5.3 (#1316)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/rommapp/romm docker tag to v4.8.1 (#1315)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v2.8.0 (#1318)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/fosrl/newt docker tag to v1.11.0 (#1319)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update kitchenowl to v0.7.7 (#1317)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Tarow committed:
feat(shelfmark): add oidc options

Tarow committed:
docs: replace homepage demo

renovate[bot] committed:
chore(deps): update ghcr.io/dgtlmoon/changedetection.io docker tag to v0.54.8 (#1320)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/filebrowser/filebrowser docker tag to v2.63.0 (#1321)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Tarow committed:
fix(trek): set OIDC_SCOPE explicit (https://github.com/mauriceboe/TREK/issues/394)

Tarow committed:
fix(trek): set app_url

Tarow committed:
fix: import reference

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v2.8.1 (#1322)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v2.8.4 (#1323)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/stirlingtools/stirling-pdf docker tag to v2.9.2 (#1324)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/filebrowser/filebrowser docker tag to v2.63.1 (#1325)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update lscr.io/linuxserver/prowlarr docker tag to v2.3.5 (#1326)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): lock file maintenance (#1327)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/alam00000/bentopdf-simple docker tag to v2.8.2 (#1329)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v2.9.0 (#1332)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update lscr.io/linuxserver/jellyfin docker tag to v10.11.8 (#1331)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v2.9.8 (#1333)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v2.9.10 (#1334)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/linuxserver/healthchecks docker tag to v4.1.1-ls340 (#1335)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/gotenberg/gotenberg docker tag to v8.30.0 (#1337)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update dependency pagefind to v1.5.0 (#1336)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/deluan/navidrome docker tag to v0.61.1 (#1328)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update beszel to v0.18.7 (#1330)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Tarow committed:
fix(shelfmark): default auth method

renovate[bot] committed:
chore(deps): update docker.io/gotenberg/gotenberg docker tag to v8.30.1 (#1338)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/arabcoders/watchstate docker tag to v1.5.0 (#1339)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.n8n.io/n8nio/n8n docker tag to v2.16.0 (#1340)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v2.9.11 (#1341)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/prom/prometheus docker tag to v3.11.1 (#1342)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/amir20/dozzle docker tag to v10.3.0 (#1343)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/autobrr/qui docker tag to v1.16.0 (#1344)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update immich to v2.7.0 (#1345)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/traefik docker tag to v3.6.13 (#1346)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update kaneo to v2.6.0 (#1347)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update immich to v2.7.2 (#1348)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/viren070/aiostreams docker tag to v2.27.0 (#1351)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/autobrr/qui docker tag to v1.16.1 (#1349)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/prom/alertmanager docker tag to v0.32.0 (#1350)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/authelia/authelia docker tag to v4.39.17 (#1352)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v2.9.12 (#1353)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update immich to v2.7.3 (#1355)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/arabcoders/watchstate docker tag to v1.5.1 (#1354)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/authelia/authelia docker tag to v4.39.18 (#1356)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/tandoorrecipes/recipes docker tag to v2.6.5 (#1357)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update immich to v2.7.4 (#1358)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/kimai/kimai2 docker tag to apache-2.53.0 (#1359)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/filebrowser/filebrowser docker tag to v2.63.2 (#1360)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/tandoorrecipes/recipes docker tag to v2.6.6 (#1362)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v2.9.13 (#1361)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/gtsteffaniak/filebrowser docker tag to v1.3.4 (#1363)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/home-assistant/home-assistant docker tag to v2026.4.2 (#1364)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): lock file maintenance (#1365)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/amir20/dozzle docker tag to v10.3.1 (#1366)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/authelia/authelia docker tag to v4.39.19 (#1367)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/drytrix/timetracker docker tag to v5.3.0 (#1369)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update dependency pagefind to v1.5.2 (#1368)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/dani-garcia/vaultwarden docker tag to v1.35.5 (#1370)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update kaneo to v2.6.1 (#1371)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/dani-garcia/vaultwarden docker tag to v1.35.6 (#1373)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/linuxserver/healthchecks docker tag to v4.1.1-ls341 (#1374)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/immich-app/immich-server docker tag to v2.7.5 (#1376)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update kaneo to v2.6.5 (#1378)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/seerr-team/seerr docker tag to v3.1.1 (#1377)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/prom/prometheus docker tag to v3.11.2 (#1375)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/immich-app/immich-machine-learning docker tag to v2.7.5 (#1380)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update kaneo to v2.6.8 (#1381)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.n8n.io/n8nio/n8n docker tag to v2.17.0 (#1382)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/sysadminsmedia/homebox docker tag to v0.25.0 (#1379)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/deluan/navidrome docker tag to v0.61.2 (#1372)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Tarow committed:
feat: reactive resume stack (#911)

* feat: reactive resume stack

* chore: update image

* chore: update image

* fix: printer app url

renovate[bot] committed:
chore(deps): update ghcr.io/amruthpillai/reactive-resume docker tag to v5.0.17 (#1383)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/mealie-recipes/mealie docker tag to v3.15.0 (#1384)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/amir20/dozzle docker tag to v10.3.2 (#1386)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/grafana/alloy docker tag to v1.15.1 (#1388)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/dani-garcia/vaultwarden docker tag to v1.35.7 (#1389)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/grafana/grafana docker tag to v13 (#1391)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/dgtlmoon/changedetection.io docker tag to v0.54.9 (#1390)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/donetick/donetick docker tag to v0.1.75 (#1387)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update actions/upload-pages-artifact action to v5 (#1385)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/amruthpillai/reactive-resume docker tag to v5.0.18 (#1392)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/tandoorrecipes/recipes docker tag to v2.6.7 (#1393)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/mealie-recipes/mealie docker tag to v3.15.1 (#1394)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/paperless-ngx/paperless-ngx docker tag to v2.20.14 (#1397)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/amir20/dozzle docker tag to v10.3.3 (#1396)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.n8n.io/n8nio/n8n docker tag to v2.17.1 (#1399)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v2.9.14 (#1398)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/drytrix/timetracker docker tag to v5.3.1 (#1400)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.n8n.io/n8nio/n8n docker tag to v2.17.2 (#1402)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/mealie-recipes/mealie docker tag to v3.15.2 (#1403)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/tandoorrecipes/recipes docker tag to v2.6.8 (#1404)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/kimai/kimai2 docker tag to apache-2.54.0 (#1405)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/seerr-team/seerr docker tag to v3.2.0 (#1407)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/fccview/jotty docker tag to v1.22.0 (#1406)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/tandoorrecipes/recipes docker tag to v2.6.9 (#1410)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/viren070/aiostreams docker tag to v2.27.1 (#1411)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/amir20/dozzle docker tag to v10.4.0 (#1412)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/kimai/kimai2 docker tag to apache-2.55.0 (#1413)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/grafana/grafana docker tag to v13.0.1 (#1414)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/gotenberg/gotenberg docker tag to v8.31.0 (#1415)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/home-assistant/home-assistant docker tag to v2026.4.3 (#1416)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/mealie-recipes/mealie docker tag to v3.16.0 (#1418)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/amir20/dozzle docker tag to v10.4.1 (#1417)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/adguard/adguardhome docker tag to v0.107.74 (#1401)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/zhfahim/anchor docker tag to v0.11.0 (#1408)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update codeberg.org/forgejo/forgejo docker tag to v15 (#1409)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/bellamy/wallos docker tag to v4.8.1 (#1419)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/autobrr/qui docker tag to v1.17.0 (#1421)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/alam00000/bentopdf-simple docker tag to v2.8.3 (#1420)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/bellamy/wallos docker tag to v4.8.2 (#1422)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/gtsteffaniak/filebrowser docker tag to v1.3.5 (#1423)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/norishapp/norish docker tag to v0.18.1 (#1395)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/usememos/memos docker tag to v0.27.0 (#1424)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/viren070/aiostreams docker tag to v2.27.2 (#1426)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/usememos/memos docker tag to v0.27.1 (#1427)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Tarow committed:
chore: norish image update

renovate[bot] committed:
chore(deps): update ghcr.io/advplyr/audiobookshelf docker tag to v2.33.2 (#1429)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.n8n.io/n8nio/n8n docker tag to v2.17.3 (#1431)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/linuxserver/healthchecks docker tag to v4.1.1-ls342 (#1432)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/gtsteffaniak/filebrowser docker tag to v1.3.6 (#1433)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.n8n.io/n8nio/n8n docker tag to v2.18.0 (#1434)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/alam00000/bentopdf-simple docker tag to v2.8.4 (#1435)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/dgtlmoon/changedetection.io docker tag to v0.54.10 (#1430)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/binwiederhier/ntfy docker tag to v2.22.0 (#1436)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update kaneo to v2.6.9 (#1437)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.n8n.io/n8nio/n8n docker tag to v2.18.1 (#1438)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/viren070/aiostreams docker tag to v2.27.3 (#1441)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/traefik docker tag to v3.6.14 (#1439)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/grimmory-tools/grimmory docker tag to v2.3.1 (#1440)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.n8n.io/n8nio/n8n docker tag to v2.18.2 (#1444)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/grafana/alloy docker tag to v1.16.0 (#1445)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/gotson/komga docker tag to v1.24.4 (#1446)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.n8n.io/n8nio/n8n docker tag to v2.18.3 (#1447)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/drytrix/timetracker docker tag to v5.3.2 (#1448)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/stirlingtools/stirling-pdf docker tag to v2.10.0 (#1449)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/home-assistant/home-assistant docker tag to v2026.4.4 (#1450)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/outlinewiki/outline docker tag to v1.7.0 (#1451)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/dani-garcia/vaultwarden docker tag to v1.35.8 (#1452)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/drytrix/timetracker docker tag to v5.4.0 (#1453)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/gtsteffaniak/filebrowser docker tag to v1.3.7 (#1454)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/arabcoders/watchstate docker tag to v1.6.0 (#1455)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): lock file maintenance (#1456)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): lock file maintenance (#1458)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/amruthpillai/reactive-resume docker tag to v5.0.19 (#1457)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update dependency vitepress-plugin-pagefind to v0.4.20 (#1459)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v3 (#1442)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/grimmory-tools/grimmory docker tag to v3 (#1443)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update kitchenowl to v0.7.8 (#1461)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/bellamy/wallos docker tag to v4.8.3 (#1460)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/norishapp/norish docker tag to v0.18.1 (#1428)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/amir20/dozzle docker tag to v10.5.0 (#1463)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v3.0.9 (#1462)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v3.0.10 (#1464)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/paperless-ngx/paperless-ngx docker tag to v2.20.15 (#1466)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update module github.com/maxlerebourg/crowdsec-bouncer-traefik-plugin to v1.6.0 (#1468)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.n8n.io/n8nio/n8n docker tag to v2.18.4 (#1469)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/amruthpillai/reactive-resume docker tag to v5.0.20 (#1470)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/kimai/kimai2 docker tag to apache-2.56.0 (#1472)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/prom/prometheus docker tag to v3.11.3 (#1471)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/drytrix/timetracker docker tag to v5.5.0 (#1477)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/freikin/dawarich docker tag to v1.7.0 (#1474)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/bellamy/wallos docker tag to v4.8.4 (#1473)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/advplyr/audiobookshelf docker tag to v2.34.0 (#1475)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/arabcoders/watchstate docker tag to v1.7.0 (#1476)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/viren070/aiostreams docker tag to v2.28.0 (#1478)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/fosrl/newt docker tag to v1.12.0 (#1480)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v3.0.11 (#1479)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.n8n.io/n8nio/n8n docker tag to v2.19.0 (#1481)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/fccview/jotty docker tag to v1.23.0 (#1482)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/fosrl/newt docker tag to v1.12.1 (#1483)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v3.0.12 (#1485)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/freikin/dawarich docker tag to v1.7.1 (#1484)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/fosrl/newt docker tag to v1.12.2 (#1486)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/drytrix/timetracker docker tag to v5.5.1 (#1488)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.n8n.io/n8nio/n8n docker tag to v2.19.1 (#1487)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/freikin/dawarich docker tag to v1.7.2 (#1489)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/fosrl/newt docker tag to v1.12.3 (#1493)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/drytrix/timetracker docker tag to v5.5.2 (#1492)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/traefik docker tag to v3.6.15 (#1491)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/gtsteffaniak/filebrowser docker tag to v1.3.8 (#1494)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/norishapp/norish docker tag to v0.18.2 (#1495)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/prom/alertmanager docker tag to v0.32.1 (#1490)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/dgtlmoon/changedetection.io docker tag to v0.55.3 (#1467)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/usememos/memos docker tag to v0.28.0 (#1465)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/amir20/dozzle docker tag to v10.5.1 (#1496)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/gotenberg/gotenberg docker tag to v8.32.0 (#1497)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Tarow committed:
refactor: grimmory icon

Tarow committed:
docs: update readme

renovate[bot] committed:
chore(deps): update ghcr.io/lldap/lldap docker tag to v2026-04-30-alpine-rootless (#1498)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/qdm12/ddns-updater docker tag to v2.10.0 (#1500)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v3.0.13 (#1499)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/calibrain/shelfmark-lite docker tag to v1.2.2 (#1502)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.n8n.io/n8nio/n8n docker tag to v2.19.2 (#1501)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update kaneo to v2.7.0 (#1504)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/calibrain/shelfmark-lite docker tag to v1.2.3 (#1503)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/grimmory-tools/grimmory docker tag to v3.0.3 (#1505)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/danielbrendel/hortusfox-web docker tag to v5.8 (#1506)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/viren070/aiostreams docker tag to v2.29.0 (#1508)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/freikin/dawarich docker tag to v1.7.3 (#1507)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): lock file maintenance (#1509)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/freikin/dawarich docker tag to v1.7.4 (#1510)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/dani-garcia/vaultwarden docker tag to v1.36.0 (#1511)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update docker.io/mauriceboe/trek docker tag to v3.0.14 (#1512)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/viren070/aiostreams docker tag to v2.29.1 (#1513)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update ghcr.io/viren070/aiostreams docker tag to v2.29.2 (#1514)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

pitschi committed:
feat(tsbridge): add tsbridge module

Adds a new tsbridge module that integrates Tailscale tsbridge as a reverse
proxy for services. Extends the homepage module to include the tsbridge
serviceHost in HOMEPAGE_ALLOWED_HOSTS when tsbridge is enabled.
